third_party/challenge_bypass_ristretto_ffi/BUILD.gn

```diff
@@ -13,6 +13,10 @@ rust_ffi("challenge_bypass_ristretto_ffi") {
 
   deps = [ "//base" ]
 
+  if (is_clang) {
```

Comment:
```
configs -= [ "//build/config/compiler:no_chromium_code" ]
configs += [ "//build/config/compiler:no_chromium_code" ]
```
? Not a huge deal though since this stuff will change soon with my PR

Reply:
I'm not sure if I get what you mean here. Isn't the snipped you've produced removing, and then adding back the same config?

Reply:
Ah.. I get what you mean now. Done.

Reply:
I'm actually having to revert it back to the warning, as those configs you mention above won't work with the template `rust_ffi`.